vllm/attention/backends/torch_sdpa.py

```diff
@@ -13,9 +13,12 @@
 
 if is_cpu():
     try:
+        from vllm._ipex_ops import ipex_ops
         from vllm.attention.ops.ipex_attn import PagedAttention
+        use_ipex = True
     except ImportError:
         from vllm.attention.ops.paged_attn import PagedAttention
+        use_ipex = False
```

Comment:
In which case is this False?

Reply:
if ipex is not installed, this will be False and fall back to torch sdpa impl.